### PR TITLE
Add agile question about sprint cancelling

### PR DESCRIPTION
**Pull request by YuriDevAT**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/agile-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,13 @@
 const agileQuiz = [
+  {
+    Question: "When might a Sprint be abnormally cancelled?",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:3]:
```suggestion
    Question: "In Agile, when might a Sprint be abnormally cancelled?",
```

File: src/data/agile-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+    Answer: "When the Sprint Goal becomes obsolete.",
+    Distractor1: "When it becomes clear that not everything will be finished by the end of the Sprint.",
+    Distractor2: "When the Developers feel that the work is too hard.",
+    Distractor3: "When the sales department has an important new opportunity.",
+    Explanation: "A Sprint could be cancelled if the Sprint Goal becomes obsolete. Only the Product Owner has the authority to cancel the Sprint.",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:8]: Please avoid using direct quotes from 3rd party resources. It is always best to rewrite it in your own words. 

```suggestion
    Explanation: "The Product Owner can cancel a Sprint if the Sprint Goal becomes obsolete.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/agile-quiz.ts (hunk 1 above)
Comment (YuriDevAT) [reply src/data/agile-quiz.ts:3]: Thanks for your suggestions.

File: src/data/agile-quiz.ts (hunk 2 above)
Comment (YuriDevAT) [reply src/data/agile-quiz.ts:8]: I see. Thanks for the advice.

**Comment by YuriDevAT** [comment]
I implemented your suggested changes. Thank you for guiding me.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**